docs/en/ingest-management/fleet/add-fleet-server-mixed.asciidoc

```diff
@@ -0,0 +1,184 @@
+. After creating the {fleet-server} policy, navigate to the policy itself and click **Add integration**.
+. Search for and select the **{fleet-server}** integration.
+. Then click **Add {fleet-server}**.
++
+// image
+. Configure the {fleet-server}:
+.. Expand **Change default**. Because you are deploying this {fleet-server} on-premises,
+you need to enter the _Host_ address and _Port_ number, `8220`.
+(In our example the {fleet-server} will be installed on the host `10.128.0.46`.)
+.. It's recommended that you also enter the _Max agents_ you intend to support with this {fleet-server}.
+This can also be modified at a later stage.
+This will allow the {fleet-server} to handle the load and frequency of updates being sent to the agent
+and ensure a smooth operation in a bursty environment.
+
+[discrete]
+[[fleet-server-add-agents]]
+== Add {fleet-server}s
+
+// What is it / why do you need it?
+Now that the policy exists, you can add {fleet-server}s.
+A {fleet-server} is an {agent} that is enrolled in a {fleet-server} policy.
+The policy configures the agent to operate in a special mode to serve as a {fleet-server} in your deployment.
+
+// How do you set it up?
+To add a {fleet-server}:
+
+. Return to the {fleet-server} policy. If you've navigated away from the page, here's how you can get there:
+.. In {kib} go to **{fleet}**.
+.. Switch to the **Agent policies** tab.
+.. Select the policy you created previously.
+
+. Click **Add agent** and follow the instructions to add a {fleet-server}.
+.. Choose the policy name for this deployment.
+.. Choose **Production** as your deployment model.
++
+Production mode is the fully secured mode where TLS certificates ensure a secure communication between {fleet-server} and {es}.
+.. Select the {fleet-server} host that was identified earlier. Click **Add host**.
+.. A **Service Token** is required so the {fleet-server} can write data to the connected {es} instance.
+Click **Generate service token** and copy the generated token.
+.. Copy the installation instructions provided in {kib}, which include some of the known deployment parameters.
+.. Replace the value of the `--certificate-authorities` parameter with your <<add-fleet-server-mixed-prereq,CA certificate>>.
+
+After {fleet-server} is installed and enrolled in {fleet},
+the newly created {fleet-server} policy is applied.
+You can see this on the {fleet-server} policy page:
+
+// image
+
+The {fleet-server} agent will also show up on the main {fleet} page as another agent
+whose life-cycle can be managed (like other agents in the deployment):
+
+// image
+
+[discrete]
+[[fleet-server-install-agents]]
+== Next steps
+
+Now you're ready to add {agent}s to your host systems. To learn how, see
+<<install-fleet-managed-elastic-agent>>.
+
+// Do we need the content below or can we use the content above to
+// point to the same page we point to for the other approaches?
```

Comment:
Take a look at the content on how to install agents above and below this comment.